tensorflow/core/framework/device_factory.h

```diff
@@ -126,8 +129,10 @@ class Registrar {
   // GPUCompatibleCPU: 70
   // ThreadPoolDevice: 60
   // Default: 50
+  // Subdevice type is empty for those devices registered
+  // by Registrar (first party devices);
   explicit Registrar(const std::string& device_type, int priority = 50) {
-    DeviceFactory::Register(device_type, new Factory(), priority);
+    DeviceFactory::Register(device_type, string(""), new Factory(), priority);
```

Comment:
Can `string("")` just be `""`?  Also, please use `/*device_type=*/device_type` and `/*subdevice_type=*/""`, these parameters get checked against named the actual parameters by our linter internally.

Reply:
done